Test/lit.site.cfg

```diff
@@ -163,21 +158,19 @@ else:
     config.available_features = [ 'ubuntu', 'posix' ]
 
 # Sanity check: Check solver executable is available
-z3Path = None
+solverRoots = os.pathsep.join(
+    (path.dirname(bareDafnyExecutable), binaryDir, os.environ["PATH"])
+)
 
-potentialSolverRoots = [path.dirname(bareDafnyExecutable), binaryDir]
-potentialSolverExecutables = ['cvc4.exe', path.join("z3", "bin", "z3"), 'z3.exe', path.join("z3", "bin", "z3.exe")]
+print(solverRoots)
 
-for root in potentialSolverRoots:
-    for executable in potentialSolverExecutables:
-        if not z3Path:
-            potentialPath = path.join(root, executable)
-            if path.exists(potentialPath):
-                z3Path = potentialPath
+solverPath = \
+    lit.util.which("z3", solverRoots) or \
+    lit.util.which("cvc4", solverRoots)
```

Comment:
I wonder how long `cvc4` has been in here. Recent versions of Dafny and Boogie don't work with it (though Boogie often works with `cvc5`). It's probably not urgent to change this, but we may want to eventually.